packages/components/src/utils/get-focusable-children.ts

```diff
@@ -22,6 +22,7 @@ const focusDisablingSelector = `:where(${[
   'details:not([open]) > *:not(details > summary:first-of-type)',
   'details:not([open]) > *:not(details > summary:first-of-type) *',
   '[tabindex^="-"]',
+  '[role=menuitem]:has([aria-current="true"]) *',
```

Comment:
This function is an helper therefore it should remain generic. Can you update the function to check if the element is visible or not using the `getComputedStyles` function for example?